pkgs/servers/sql/pgcat/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pgcat";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "postgresml";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-ESHBOh9JSzu6Zxh0z/+nebumi/zyFVdTK0DIwR/46Xo=";
```

Comment:
This is optional since it could be automatized treewide at some point.

The idea is to avoid using `sha256` as attribute and use `hash` everywhere.

Since there could be different hashing algorithms in use, this make sense.